### PR TITLE
fixed a few german typos

### PR DESCRIPTION
**Pull request by chucktheripper**
Hey there, I ran the nightly build and encountered some german typos.

Thanks for all your work!

### CODE REVIEW

File: Goldleaf/Program/RomFs/Language/Strings/de.json [hunk 1]
```diff
@@ -159,7 +159,7 @@
     "Diese NSP wurde empfangen:",
     "Soll diese installiert werden?",
     "Durchschnittliche Geschwindigkeit:",
-    "Die USB Installation erfolgreich durchgeführt. Der PC Client kann geschlossen werden.",
```
Comment (lookye) [inline Goldleaf/Program/RomFs/Language/Strings/de.json]: Kann man machen, braucht man nicht.
You can do this, but "wurde" is not needed necessary.

File: Goldleaf/Program/RomFs/Language/Strings/de.json [hunk 2]
```diff
@@ -225,36 +225,36 @@
...
     "Soll die Konsole neustarten oder ausgeschaltet werden?",
-    "Datei oder Ordner wurde kopiert. Die Zwischenable wurde aktualsiert.",
+    "Datei oder Ordner wurde kopiert. Die Zwischenablage wurde aktualsiert.",
     "Neustart",
     "Ausschalten",
     "Ok",
     "Titel Key;",
     "Signatur Typ:",
     "Ordnergröße:",
-    "Wähle einen neuen Namen um die umzubenennen.",
```
Comment (lookye) [inline Goldleaf/Program/RomFs/Language/Strings/de.json]: It should be:
"Wähle einen neuen Namen." or Wähle einen neuen Namen um die Datei umzubenennen."

File: Goldleaf/Program/RomFs/Language/Strings/de.json (hunk 1 above)
Comment (chucktheripper) [reply Goldleaf/Program/RomFs/Language/Strings/de.json]: Doesn't seem to be valid german to me without the "wurde", I'd remove the "Die" or add the "wurde".

Comment (lookye) [reply Goldleaf/Program/RomFs/Language/Strings/de.json]: Yeah, best would be to remove "Die" at the beginning (sorry had a long day).

Comment (chucktheripper) [reply Goldleaf/Program/RomFs/Language/Strings/de.json]: Looks better now 👍

**Comment by XorTroll** [comment]
Good to merge it now?

**Pull request merged**